### PR TITLE
Part1

### PR DESCRIPTION
**Pull request by Sonder1966**
Part 1

### CODE REVIEW

**Comment by Sonder1966** [comment]
Part 1

**Review by matkatmusic — APPROVED** [review]

**Comment by Sonder1966** [comment]
Part 1b/5

File: main.cpp [hunk 1]
```diff
@@ -13,75 +13,104 @@ What you create in this project will be used as the basis of Project 5 in the co
...
+ */
+
+/*
+Thing 1) DAW
+5 properties:
+    1) stores digital data
+    2) prompts user input
+    3) hosts plugins
+    4) automates process
+    5) generates sound
```
Comment (matkatmusic) [inline main.cpp]: These are actions, not properties!
Choose properties, like "number of channels allowed" or "number of plugins included"
- [x] revise

File: main.cpp [hunk 2]
```diff
@@ -13,75 +13,104 @@ What you create in this project will be used as the basis of Project 5 in the co
...
+    2) process audio
+    3) mix music
+ */
+
+/*
+Thing 2) airport
+5 properties:
+    1) number of planes
+    2) employs floor satff
+    3) service aeroplanes
```
Comment (matkatmusic) [inline main.cpp]: These are actions, not properties!
- [ ] revise

File: main.cpp [hunk 3]
```diff
@@ -13,75 +13,104 @@ What you create in this project will be used as the basis of Project 5 in the co
...
+    4) number of runways
+    5) tracks with radar
```
Comment (matkatmusic) [inline main.cpp]: `tracks` is a verb, not a property
- [ ] revise

File: main.cpp [hunk 4]
```diff
@@ -13,75 +13,104 @@ What you create in this project will be used as the basis of Project 5 in the co
...
+    2) centralise travel departure
+    3) service travel industry
+ */
+
+/*
+Thing 3) farm
+5 properties:
+    1) number of chickens
+    2) number of cows
+    3) single farmer
```
Comment (matkatmusic) [inline main.cpp]: `single farmer` is not a property of a farm.   it's a thing all by itself.
pick something else, like "number of workers"
- [ ] revise

File: main.cpp [hunk 5]
```diff
@@ -13,75 +13,104 @@ What you create in this project will be used as the basis of Project 5 in the co
...
+ */
+
+/*
+Thing 4) school
+5 properties:
+    1) number of students
+    2) number of teachers
+    3) hours per week of teaching
+    4) hours per week of lunch
+    5) separate classromms
```
Comment (matkatmusic) [inline main.cpp]: `separate classrooms` is not a property, it's a thing. 
pick something else, like `average number of blackboards per teacher'
- [ ] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]

**Review by matkatmusic — APPROVED** [review]

File: main.cpp [hunk 6]
```diff
@@ -113,7 +145,19 @@ Thing 4) school
...
 
-
+/*
+Thing 10) audio compressor
+5 properties:
+    1)input
+    2)output
+    3)make up gain
+    4)threshold
+    5)ratio
```
Comment (matkatmusic) [inline main.cpp]: Are you sure you can extract 5 properties and 3 actions out of EACH of these?
What are 5 properties of `make-up gain`? 
- [ ] revise.  Consider picking a different Thing for 10)

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 6 above)
Comment (Sonder1966) [reply main.cpp]: New Thing: Truck?

**Review by matkatmusic — APPROVED** [review]

File: main.cpp [hunk 7]
```diff
@@ -145,6 +130,78 @@ Thing 4) school
...
+ */
+
+/*
+Thing 9) fuel system
+5 properties:
+    1) fuel injection pressure
+    2) fuel type
+    3) fuel capacity
+    4) fuel filter amount
+    5) fuel pump power
```
Comment (matkatmusic) [inline main.cpp]: the word `fuel` is redundant here.   you can eliminate this word from your 5 properties
- [ ] Revise it in the next part.

**Review by matkatmusic — APPROVED** [review]

**Review by matkatmusic — CHANGES_REQUESTED** [review]
- Type names need to begin with a Capital letter. All of your type names begin with a lower case letter and they don't use `CamelCaseIfTheNameUsesMultipleWords`
- `(` and `)` should not have spaces around them.  consult the coding standard in the Readme.MD
- `=` should have spaces around it.  some of yours do not
- [ ] revise

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**